features/nationaliteit.feature

```diff
@@ -29,146 +29,138 @@ Functionaliteit: Bepalen van de actuele nationaliteit van een persoon
-      | 1      | 4         |                       | 310                   |                          | V                                  |                 | 19750615                        |
+    Gegeven de persoon met burgerservicenummer '000000036' heeft een 'nationaliteit' met de volgende gegevens
+      | reden opnemen (63.10) | bijzonder Nederlanderschap (65.10) | datum ingang geldigheid (85.10) |
+      | 310                   | V                                  | 19750615                        |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                              |
       | type                | RaadpleegMetBurgerservicenummer     |
-      | burgerservicenummer | 999994748                           |
-      | fields              | burgerservicenummer,nationaliteiten |
-      Dan heeft de persoon met burgerservicenummer '999994748' een nationaliteit met alleen de volgende gegevens
+      | burgerservicenummer | 000000036                           |
+      | fields              | nationaliteiten |
+      Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
       | naam                                | waarde   |
       | aanduidingBijzonderNederlanderschap | V        |
       | redenOpname                         | 310      |
       | datumIngangGeldigheid               | 19750615 |
 
     @gba
     Scenario: de persoon heeft een onbekende nationaliteit
-      Gegeven de persoon met burgerservicenummer '999993367' heeft de volgende gegevens
-      | Stapel | Categorie | nationaliteit (05.10) | reden opnemen (63.10) | reden beëindigen (64.10) | bijzonder Nederlanderschap (65.10) | onjuist (84.10) | datum ingang geldigheid (85.10) |
-      | 1      | 4         | 0000                  | 311                   |                          |                                    |                 | 00000000                        |
+      Gegeven de persoon met burgerservicenummer '000000048' heeft een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
+      | 0000                  | 311                   | 00000000                        |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                              |
       | type                | RaadpleegMetBurgerservicenummer     |
-      | burgerservicenummer | 999993367                           |
-      | fields              | burgerservicenummer,nationaliteiten |
-      Dan heeft de persoon met burgerservicenummer '999993367' een nationaliteit met alleen de volgende gegevens
+      | burgerservicenummer | 000000048                           |
+      | fields              | nationaliteiten |
+      Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
       | naam                  | waarde   |
       | nationaliteit         | 0000     |
```

Comment:
schrijven we deze niet als code en omschrijving?

zelfde geldt hieronder voor redenOpname

Reply:
Aangepast